bigframes/core/pyformat.py

```diff
@@ -51,9 +57,13 @@ def _field_to_template_value(
     if isinstance(value, table_types):
         return _table_to_sql(value)
 
-    # TODO(tswast): convert pandas DataFrame objects to gbq tables or a literals subquery.
+    if isinstance(value, pandas.DataFrame):
+        # TODO: create bigframes DataFrame. Need a Session.
```

Comment:
TODO is out of date.

```suggestion
        # Use the _deferred engine to avoid loading data too often during dry run.
```